newrelic-agent/src/main/java/com/newrelic/agent/tracers/DefaultTracer.java

```diff
@@ -854,7 +854,9 @@ private void recordMessageBrokerMetrics(MessageConsumeParameters messageConsumeP
 
     private void recordFaasAttributes(CloudParameters cloudParameters) {
         setAgentAttribute(AttributeNames.CLOUD_PLATFORM, cloudParameters.getPlatform());
```

Comment:
Is it possible that `cloudParameters.getPlatform()` could potentially cause an NPE as well?

Reply:
No because every instance of us creating CloudParameters fills the platform field